src/mscorlib/src/System/Globalization/CompareInfo.Unix.cs

```diff
@@ -87,6 +87,46 @@ internal static unsafe int IndexOfOrdinalCore(string source, string value, int s
             return -1;
         }
 
+        internal static unsafe int IndexOfOrdinalCore(ReadOnlySpan<char> source, ReadOnlySpan<char> value, bool ignoreCase)
```

Comment:
>IndexOfOrdinalCore [](start = 35, length = 18)

why duplicating the code here? isn't better to have such methods take char* and avoid the code duplication?

Reply:
If we have a method that takes a char*, we would have to pin the span/string to get the pointer in all cases (unnecessarily) instead of only around the OS call. Given this change - https://github.com/dotnet/coreclr/pull/16434, I want to re-measure the cost of casting string to span and see if we can address the code duplication that way. I will address this outside this PR.